### DISCUSSION TITLE
Can KEDA add information to the replica?

### DISCUSSION THREAD

**Discussion by buccielias24** [kedacore/keda#5170] [Q&A / Need Help] [answered]
Hi guys, I'm wondering if KEDA can add some useful information to the replica instantiated by the scaledObject, I want to control through some env value or whatever the queue where my app is going to get the messages, currently I have many queues and just 1 deploy, in order to avoid to create one deploy per queue and scaled it, I want to know if I can configure something to do that.

**Comment by tomkerkhove** [comment, marked as answer]
No, unfortunately not

**Reply by buccielias24** [reply]
Got it, I wanted to confirm, thanks for the answer !
